mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx

```diff
@@ -25,6 +26,7 @@ export const SidebarTrigger = forwardRef((props: PropsWithChildren<SidebarTrigge
       <div
         ref={ref}
         role="button"
```

Comment:
1. Task is not about adding aria-label to the Sidebar Trigger
2. Please read more about aria-label. Need research which tet should be in the aria and which tags couldn't include aria

For example, https://doka.guide/a11y/aria-label/


Please write  other oficial documentation about aria

Reply:
I think I should have added the aria tag only to the theme toggle button. The text value should match the language of the application.
Also I need to research and write for which elements we should add arria label or text values for accessibility.
Am I thinking in the right direction ?

Reply:
> I think I should have added the aria tag only to the theme toggle button.
Yes

Reply:
> The text value should match the language of the application.

Need research. It's a task to read  in the docs which value  there should be

Reply:
> Also I need to research and write for which elements we should add arria label or text values for accessibility.

Research need all time to check issues (maybe it's not a bug or bug is deprecated and etc.

Reply:
My research: 
https://docs.google.com/document/d/1pBUqgu7fRuseeu_52bSyRhO3oeVFuPvZ7gwjV_h8Wvw/edit?usp=sharing